console/src/helpers/versionUtils.js

```diff
@@ -6,7 +6,7 @@ export const READ_ONLY_RUN_SQL_QUERIES = 'readOnlyRunSqlQueries';
 
 // list of feature launch versions
 const featureLaunchVersions = {
-  // feature: 'v1.0.0'
+  // feature: 'v1.0.1'
```

Comment:
CMIAW. This feature need not have semver checks since it'll go out in RC?

Reply:
Yep. No version checks needed